skin.scss

```diff
@@ -0,0 +1,35 @@
+.highlight {
+    background-color: #ddd;
+}
+
+button.btn {
+  border-radius: 10px;
+  border-style: solid;
+  border-width: 20px;
+  font-weight: normal;
+  line-height: 1.15;
+  text-align: center;
+  vertical-align: middle;
+}
+btn{ 

```

Comment:
This isn't quite correct. You can add a `.` at the beginning of your selector so that it becomes `.btn`, or add the button element selector so that it becomes `button.btn`.

Personally I would actually move this to be nested within the `button.btn` ruleset. So it would look like:

```scss
button.btn {
  border-radius: 10px;
  // other declarations here...
  &--primary {
    background-color: #ffffff;
  }
  &--secondary {
    background-color: #5a1212;
  }
  // etc
}
```

This keeps the `.btn` styles with their variants (`.btn--primary`). It makes it more obvious to the reader that they're related.

Reply:
done it.

Reply:
Cool. If you push your branch to Github again then it should update the PR with your new commits